CMakeLists.txt

```diff
@@ -36,6 +36,30 @@ conan_cmake_run(CONANFILE conanfile.py
         steam=${STORM_ENABLE_STEAM}
 )
 
+if (NOT WIN32)
```

Comment:
Can we put it into our conan artifactory instead?

Reply:
Yes it is possible. I don't know conan very well and I think that it overcomplicated. If you can do it - I'm ready to add it, if not - then I would leave this part as it is, because conan simplifies building under Windows, but not under Linux.